.dockerignore

```
@@ -0,0 +1,503 @@
+### Flask ###
+instance/*
+!instance/.gitignore
+.webassets-cache
+#.env
```

We should ignore this if we're adding `env_file` in docker-compose.yml. Also, I think we should move this to the bottom of the file along with the other new entries 🙂 
```suggestion
.env
```